mon-pix/app/controllers/campaigns/campaign-landing-page.js

```diff
@@ -1,8 +1,13 @@
 import { action } from '@ember/object';
 import Controller from '@ember/controller';
+import ENV from 'mon-pix/config/environment';
 
 export default class CampaignLandingPageController extends Controller {
 
+  get isNotPixConcours() {
```

Comment:
```suggestion
  get isNotPixContest() {
```
**Question :** Ne faudrait-il pas éviter de mélanger les langues dans le nommage ?